modules/wallet.go

```diff
@@ -207,7 +207,7 @@ type (
 		// builder. Items are returned by index.
 		ViewAdded() (newParents, siacoinInputs, siafundInputs, transactionSignatures []int)
 
-		// Drop indicates that a transaction is no longer useful, will not be
+		// Drop indicates that a transaction is no longer useful and will not be
```

Comment:
the first two items have the same subject so they should not be separated by a comma, e.g. "I went to the store and picked up some apples" vs. "I went to the store, and picked up some apples"